### ISSUE TITLE
az cli "hybrid connections not supported on linux app."

### ISSUE THREAD

**Issue by JasperCGK** [Azure/azure-cli#17124]
**Describe the bug**
Since hybrid-connections are now supported on Linux webapps via the portal, it should also be supported via CLI,
The following warning message appears: "hybrid connections not supported on linux app." and it does not add the hybrid-connection.

**To Reproduce**
execute command on linux webapp: 
az webapp hybrid-connection add --hybrid-connection
                                --name
                                --namespace
                                --resource-group
                                [--slot]
                                [--subscription]

**Expected behavior**
A hybrid connection should be added to the linux webapp

**Environment summary**
```
Linux-4.15.0-1106-azure-x86_64-with-debian-10.2 (Cloud Shell)
Python 3.6.10
Installer: DEB

azure-cli 2.19.0 *
```

**Additional context**
Documentation does not mention that this does not work on linux webapps

**Comment by yungezz** [comment]
route to appropriate team

**Comment by WibblyDibbler** [comment]
This was marked as resolved by #17146, which was released in version 2.23.0 (according to the release notes).  After upgrading to that version, this error persists.

>az --version
azure-cli                         2.23.0

core                              2.23.0
telemetry                          1.0.6
